lib/ts/recipe/passwordless/components/features/linkClickedScreen/index.tsx

```diff
@@ -45,8 +45,11 @@ const LinkClickedScreen: React.FC<PropType> = (props) => {
         const linkCode = getURLHash();
 
         if (preAuthSessionId === null || preAuthSessionId.length === 0 || linkCode.length === 0) {
-            await SuperTokens.getInstanceOrThrow().redirectToAuthWithoutRedirectToPath(undefined, props.history, {
-                error: "signin",
+            await SuperTokens.getInstanceOrThrow().redirectToAuth({
+                history: props.history,
+                queryParams: {
+                    error: "signin",
+                },
```

Comment:
need to explicitly set redirectBack to false -> internal use of this function should always explicitly set this value to prevent future bugs.